bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -114,38 +123,38 @@ if args.make_signal_kde:
                          (len(mass2), N_original, str(args.min_mass)))
     else:
         idx = numpy.full(N_original, True)
-    mass2_s = mass2
-    mass1_s = data_signal['mass1'][idx]
-    assert min(mass1_s - mass2_s) > 0
-    eta_s = data_signal['eta'][idx]
-    chi_eff_s  = data_signal['chi_eff'][idx]
-    mchirp_s = data_signal['mchirp'][idx]
+    mass1 = data_signal['mass1'][idx]
+    assert min(mass1 - mass2) > 0
+    eta = data_signal['eta'][idx]
```

Comment:
Similarly to the template parameters, you shouldn't have to evaluate eta, chi_eff, mchirp explicitly here, they should only be used if they are specified as fit_param options

Reply:
@PRAVEEN-mnl can you address this comment/request?